lib/bgsite_official_web/router.ex

```diff
@@ -79,14 +79,14 @@ defmodule BgsiteOfficialWeb.Router do
     get "/admin/register", AdminRegistrationController, :new
     post "/admin/register", AdminRegistrationController, :create
     get "/admin", AdminController, :index
+    live "/website_live/:id", WebsiteLive
```

Comment:
Well, this change has no purpose, you move a live router from `:require_authenticated_admin` to `:require_authenticated_admin`, why?